docs/distributing-an-application/create-licenses.md

```diff
@@ -20,9 +20,11 @@ If you are looking to create or manage custom license fields you can do so in th
 The name of the customer to whom this license is assigned.
 
 ## Channel (Required)
-When you create a license you'll need to assign it to a release channel (if it is a customer,
+When you create a license you'll need to assign it to at least one release channel (if it is a customer,
```

Comment:
`if it is a customer, likely channel is Stable` sounds awkward how about `Stable is a likely channel for customers,`  something along those lines.